docs/UserGuide.md

```diff
@@ -1067,44 +1070,50 @@ A: HomeBoss only supports English characters.
-| List    | `customer list`                                                                                      | `customer list`                                                                                             |
-| Find    | `customer find KEYWORD [MORE_KEYWORDS]`                                                              | `customer find Ibrahim Yu`                                                                                  |
-| Edit    | `customer edit CUSTOMER_ID [--name NAME] [--phone PHONE_NUMBER] [--email EMAIL] [--address ADDRESS]` | `customer edit 1001 --name Gabriel -–phone 1234 5678 --email gabrielrocks@gmail.com --address RVRC Block B` |
-| Delete  | `customer delete CUSTOMER_ID`                                                                        | `customer delete 1001`                                                                                      |
+| Command | Format                                                                                                                                | Examples                                                                                                                 |
+|---------|---------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| Add     | `customer add --name NAME --phone PHONE_NUMBER --email EMAIL --address ADDRESS`{.swift}                                               | `customer add --name Gabriel --phone 87654321 --email Gabrielrocks@gmail.com --address RVRC Block B`{.swift}             |
+| View    | `customer view CUSTOMER_ID`{.swift}                                                                                                   | `customer view 1`{swift}                                                                                                 |
+| List    | `customer list`{.swift}                                                                                                               | `customer list`{.swift}                                                                                                  |
+| Find    | `customer find KEYWORD [MORE_KEYWORDS]`{.swift}                                                                                       | `customer find Julius Yang`{.swift}                                                                                      |
+| Edit    | `customer edit CUSTOMER_ID [--name NAME] [--phone PHONE_NUMBER] [--email EMAIL] [--address ADDRESS]`{.swift}                          | `customer edit 1 --name Gabriel -–phone 12345678 --email Gabrielrock@gmail.com --address Block 10 Tampines Road`{.swift} |
```

Comment:
I think the dash for the example phone prefix might be different from each other

Reply:
Good catch! Have changed to the correct dash